### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by Parilia**
Updated the readme's lazy install instructions to reflect the recent changes to the configs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,15 +71,21 @@ use 'f4z3r/gruvbox-material.nvim'
...
-  priority = 1000,
-  config = function()
-    -- configs go here
-  end,
+    'f4z3r/gruvbox-material.nvim',
+	name = 'gruvbox-material',
+	lazy = false,
+	priority = 1000,
+	opts = {
+        italics = true,                       		-- enable italics in general
```
Comment (f4z3r) [inline README.md]: I don't think it is necessary to duplicate the entire configuration here. Maybe just add a comment that refers to the section below?

Also, I have the lua code to use 2 spaces for indentation (see `stylua.toml`). Can you also use this here please?

**Comment by Parilia** [comment]
Okay hopefully that fix is good

File: README.md [hunk 2]
```diff
@@ -71,15 +71,12 @@ use 'f4z3r/gruvbox-material.nvim'
...
-  name = 'gruvbox-material',
-  lazy = false,
-  priority = 1000,
-  config = function()
-    -- configs go here
-  end,
+    name = 'gruvbox-material',
+	  lazy = false,
+	  priority = 1000,
+	  opts = {},
```
Comment (f4z3r) [inline README.md]: The spacing is still strange here. There seems to be a space too many, and tabs combined ...

**Comment by Parilia** [comment]
Okay pretty sure thats fixed and its good you pointed it out as i realised i had missed something in my init file, forgot to put vim.opt.softtabstop = 2 in. I am in the process of rewriting my init

**Comment by f4z3r** [comment]
> Okay pretty sure thats fixed and its good you pointed it out as i realised i had missed something in my init file, forgot to put vim.opt.softtabstop = 2 in. I am in the process of rewriting my init

No worries. There still seems to be an indentation issue though 😄 The initial value in the table in indented by 2, and the remaining ones are indented by 4.

**Comment by Parilia** [comment]
If it wouldnt be too much trouble could you please give me an example of how you would prefer it as i dont see any issues the the indentation.

**Comment by f4z3r** [comment]
@Parilia your last commit is exactly what I meant. Thanks a lot. Looks good to me 👍🏽

**Review by f4z3r — APPROVED** [review]

**Comment by Parilia** [comment]
yea funnily enough I just sorted conform out so maybe that helped a little, im used to working in web dev so maybe my tabs are bit bigger normally.

**Pull request merged**